pr-template.md

```diff
@@ -19,19 +19,10 @@
 
 #### Reviewers
 
-**Review 1**
-- [ ] :+1:
-
-**Review 2** \*
-- [ ] :+1:
-
-**Review 3** _(optional)_
-- [ ] :+1:
+Must be minumum of 2 reviewers consisting of at least one SE, SA or Project Guru.
```

Comment:
There are Guild Leaders, Guild Members and Guild Associates as far as I remember